pandas/core/indexes/base.py

```diff
@@ -1110,13 +1110,14 @@ def to_series(self, index=None, name=None):
         return Series(self._to_embed(), index=index, name=name)
 
     def to_frame(self, index=True):
-        """Create a DataFrame with a column containing the Index.
+        """
+        Create a DataFrame with a column containing the Index.
 
         .. versionadded:: 0.21.0
 
         Parameters
         ----------
-        index : boolean (default True)
+        index : boolean, default True
```

Comment:
@jorisvandenbossche on the [docstring guide rst file](https://github.com/python-sprints/python-sprints.github.io/blob/master/pandas/guide/source/pandas_docstring.rst), the example dosctring provides: `color : str (default 'blue')`. So should it be `(default True)` or `, default True`?

Reply:
Whoops, I was using the virtual in my head :-) That's something we need to fix, we decided on `, default True` I think

Reply:
See, https://github.com/python-sprints/python-sprints.github.io/pull/62